### PR TITLE
Added info about working through material on your own including link …

### PR DESCRIPTION
**Pull request by katharinebreeze**
…to microsoft stream.

Just raising a pull request so this doesn't get buried.

### CODE REVIEW

File: rmd_files/README.Rmd [hunk 1]
```diff
@@ -45,6 +47,18 @@ If you have any problems with the above please get in touch with the course orga
 
 All the examples in the presentation/README are available in the R script example_code.R.
 
+## Remote learning
+
+Here are a few suggestions if you are going through this material in your own time:
+
+* Both the read me and slides contain the same content so you can use whichever works best for you - the only difference is the exercise solutions are only in the slides
+*  To open the slides, first clone the repo (see [pre-material](#pre-material) and then open the file "slides.html" from RStudio by clicking "View in web browser"
```
Comment (chloepugh) [inline rmd_files/README.Rmd:55]: I think a matching close bracket ")" is needed after `(#pre-material)`

Comment (katharinebreeze) [reply rmd_files/README.Rmd]: fixed! (:

**Comment by chloepugh** [comment]
Sorry also noticed that the new version of readme.md needs committing, but that can be done on the master branch

**Comment by katharinebreeze** [comment]
Yeah that was deliberate just in case there were merge conflicts! I've updated it on the master branch now.

**Pull request merged**